### DISCUSSION TITLE
Does it support the data masking function?

### DISCUSSION THREAD

**Discussion by 1gil2** [getredash/redash#5959] [Q&A]
Hi there!
We are making it easier to look up data through a redash.

I want to mask the sensitivity information.
ex) email : example@example.com -> e*****le@ex***.com

Is it possible?
Thank you.

**Comment by dgpratikpatil** [comment]
Hi
Were you able to find a  way to do this?

**Comment by justinclift** [comment]
Redash itself doesn't have functionality to mask data like that, but depending on what kind of data sources you're using *they* might do.

For example with PostgreSQL, if you have some tables with sensitive data in them (eg email address), then you could create a VIEW (in PostgreSQL) to show just the columns you want.  Then you would create a user in PostgreSQL that only has permission to see that view, not the table behind it.

Does that make sense? :smile:
